### PR TITLE
Update Conversant adapter to Prebid 1.0

### PR DESCRIPTION
**Pull request by pycnvr**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Other

## Description of change

Update Conversant adapter to Prebid 1.0

- pyang@conversantmedia.com
- [x] official adapter submission

### CODE REVIEW

**Comment by pycnvr** [comment]
The CI build seems to have failed at a unrelated spot.  Please advise if there are changes needed.  Thanks.

**Comment by mkendall07** [comment]
@pycnvr 
One other thing. I noticed when testing that the bid response back gave me a `vastUrl` that returned an empty vast document. Not sure if that's expected or not since it's a test bid but wanted to bring it up.

**Comment by pycnvr** [comment]
@mkendall07 That sounds strange.  I'll check it out.  Thanks.

File: modules/conversantBidAdapter.js [hunk 1]
```diff
@@ -115,14 +115,14 @@ export const spec = {
...
-   * 
+   *
    * @param {*} serverResponse A successful response from the server.
    * @return {Bid[]} An array of bids which were nested inside the server.
    */
   interpretResponse: function(serverResponse, bidRequest) {
     const bidResponses = [];
     const requestMap = {};
-    const currency = serverResponse.cur || 'USD';
+    serverResponse = serverResponse.body;
```
Comment (mkendall07) [inline modules/conversantBidAdapter.js]: good catch!

**Pull request merged**